src/entities/Profile.ts

```diff
@@ -77,8 +82,11 @@ export default class Profile {
 	@IsOptional()
 	public linkedin?: string;
 
+	@Column({ 'default': Visibility.PUBLIC })
```

Comment:
I think you also need to set the enum options on the decorator, e.g. here is the decorator used for the course property.

```
	@Column({
		'type': 'enum',
		'enum': Course
	})
```